sdk/core/azure-core/src/main/java/com/azure/core/http/rest/IterableResponse.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.rest;
+
+import reactor.core.publisher.Flux;
+
+/**
+ * This class provides utility to iterate over values.
+ * @param <T> value
+ */
+public class IterableResponse<T> {
```

Comment:
This class doesn't seem to do anything. Is it intended?

Reply:
Still deciding, if need separate class or not. Most probably, I will remove it .